src/Lykke.Logs/Obsolete/ILykkeLogToAzureStoragePersistenceManager.cs

```diff
@@ -1,5 +1,8 @@
+using System;
+
 namespace Lykke.Logs
 {
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
See above